src/components/FooterSection.vue

```diff
@@ -1,48 +1,50 @@
 <template>
-<footer class="footer secondary">
-  <div class="columns">
-    <div class="column">
-      <a href="https://creativecommons.org/">
-        <img class="footer_logo" alt="Footer Logo" src="../assets/cc-logo_white.png">
-      </a>
-      <div class="cc-footer-links">
-        <div class="menu-footer-links-container">
-            <ul id="menu-footer-links" class="menu">
-                <li class="menu-item menu-item-type-post_type menu-item-object-page"><a href="https://creativecommons.org/about/contact/">Contact</a></li>
-                <li class="menu-item menu-item-type-post_type menu-item-object-page"><a href="https://creativecommons.org/privacy/">Privacy</a></li>
-                <li class="menu-item menu-item-type-post_type menu-item-object-page"><a href="https://creativecommons.org/policies/">Policies</a></li>
-                <li class="menu-item menu-item-type-post_type menu-item-object-page"><a href="https://creativecommons.org/terms/">Terms</a></li>
-            </ul>
-        </div>
-    </div>
-  </div>
-    <div class="column">
-      <div class="column footer_contact">
-        <h6><a href="https://creativecommons.org/about/contact/">We'd love to hear from you!</a></h6>
-          <address>
-            Creative Commons<br>
-            PO Box 1866, Mountain View, CA 94042
-          </address>
-          <ul>
-            <li><a href="mailto:info@creativecommons.org">info@creativecommons.org</a></li>
-            <li><a href="tel:1-415-429-6753">1-415-429-6753</a></li>
+<footer class="main-footer">
+  <div class="container">
+    <div class="columns">
+      <div class="column">
+        <a href="https://creativecommons.org" class="main-logo margin-bottom-bigger">
+          <img class="footer_logo" alt="Footer Logo" src="../assets/cc-logo_white.png">
```

Comment:
I think this would be the only problem I saw... because of the footer logo style relies on SVG, so we can add the same style to `<img>` or change it to inline SVG. I would rather SVG over png because png sometimes get pixelated on 2x resolutions and impact performance because of its weight

Reply:
Sorry Hugo. I accidentally merged this PR instead of another one. I'll fix this in a separate PR